typify-cli/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "typify-cli"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[[bin]]
+name = "typify-cli"
+path = "src/main.rs"
+
+[dependencies]
+serde = "1.0"
+schemars = "0.8"
+serde_json = "1.0"
+typify = { path = "../typify" }
```

Comment:
can you add the version here as well please?

Reply:
done